tests/models/test_sasrec.py

```diff
@@ -0,0 +1,581 @@
+import typing as tp
+from typing import List
+
+import numpy as np
+import pandas as pd
+import pytest
+import torch
+from pytorch_lightning import Trainer, seed_everything
+
+from rectools.columns import Columns
+from rectools.dataset import Dataset, IdMap, Interactions
+from rectools.models.sasrec import IdEmbeddingsItemNet, SASRecDataPreparator, SASRecModel, SequenceDataset
+from tests.models.utils import assert_second_fit_refits_model
+from tests.testing_utils import assert_id_map_equal, assert_interactions_set_equal
+
+
+# Ignore pytorch_lightning warnings
+@pytest.mark.filterwarnings("ignore::pytorch_lightning.utilities.warnings.PossibleUserWarning")
+# Ignore non-empty checkpoint directory warnings, SASRec user warnings
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
You can just change to `on_supported_targets=ignore` where you don't want the warnings to be received from `recommend` method. Let's drop this filtering

Reply:
fixed